resources/eventing/charts/nats/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Default values for nats.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
```

Comment:
Why do you need a global value ?

> https://helm.sh/docs/chart_template_guide/subcharts_and_globals/
> Global values are values that can be accessed from any chart or subchart by exactly the same name. Globals require explicit declaration. You can't use an existing non-global as if it were a global.

Reply:
It was prepared for the future structure, where "eventing" will have a hierarchy of charts.
Changed for now to be "local"